Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/__init__.py

```diff
@@ -0,0 +1,135 @@
+# coding:utf-8
+#!/usr/bin/python
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+# -------------------------------------------------------------------------
+"""! @brief
+DCCsi/Tools/DCC/Maya/__init__.py
+
+This init allows us to treat Maya setup as a DCCsi tools python package
+"""
+# -------------------------------------------------------------------------
+# standard imports
+import os
+import site
+import inspect
+import traceback
+from pathlib import Path
+import logging as _logging
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# global scope
+_PACKAGENAME = 'Tools.DCC.Maya'
+
+__all__ = ['config',
+           'constants',
+           'setup',
+           'start']
+
+_LOGGER = _logging.getLogger(_PACKAGENAME)
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# Maya is frozen
+#_MODULE_PATH = Path(__file__)
+# https://tinyurl.com/y49t3zzn
+# module path when frozen
+_MODULE_PATH = Path(os.path.abspath(inspect.getfile(inspect.currentframe())))
+_LOGGER.debug('_MODULE_PATH: {}'.format(_MODULE_PATH))
+_PATH_DCCSI_TOOLS_MAYA = Path(_MODULE_PATH.parent)
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS_DCC = Path(_PATH_DCCSI_TOOLS_MAYA.parent)
+_PATH_DCCSI_TOOLS_DCC = Path(os.getenv('PATH_DCCSI_TOOLS_DCC', _PATH_DCCSI_TOOLS_DCC.as_posix()))
+site.addsitedir(_PATH_DCCSI_TOOLS_DCC.as_posix())
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSI_TOOLS = Path(_PATH_DCCSI_TOOLS_DCC.parent)
+_PATH_DCCSI_TOOLS = Path(os.getenv('PATH_DCCSI_TOOLS', _PATH_DCCSI_TOOLS.as_posix()))
+
+# we need to set up basic access to the DCCsi
+_PATH_DCCSIG = Path(_PATH_DCCSI_TOOLS.parent)
+_PATH_DCCSIG = Path(os.getenv('PATH_DCCSIG', _PATH_DCCSIG.as_posix()))
+site.addsitedir(_PATH_DCCSIG.as_posix())
+# -------------------------------------------------------------------------
+
+
+# -------------------------------------------------------------------------
+# now we have access to the DCCsi code and azpy
+import azpy.test.entry_test
+from azpy.env_bool import env_bool
+from azpy.constants import ENVAR_DCCSI_GDEBUG
+from azpy.constants import ENVAR_DCCSI_DEV_MODE
+from azpy.constants import ENVAR_DCCSI_LOGLEVEL
+from azpy.constants import ENVAR_DCCSI_GDEBUGGER
+from azpy.constants import FRMT_LOG_LONG
+
+#  global space
+_DCCSI_GDEBUG = env_bool(ENVAR_DCCSI_GDEBUG, False)
+_DCCSI_DEV_MODE = env_bool(ENVAR_DCCSI_DEV_MODE, False)
+_DCCSI_GDEBUGGER = env_bool(ENVAR_DCCSI_GDEBUGGER, 'WING')
+
+# default loglevel to info unless set
+_DCCSI_LOGLEVEL = int(env_bool(ENVAR_DCCSI_LOGLEVEL, _logging.INFO))
+if _DCCSI_GDEBUG:
```

Comment:
Did you ever resolve the multiple instances of logging configuration? I feel like all the scattered instantiated config instances of them should be pruned. It would be easier to control the logger from a single, central point. From this point, in the event that it is needed, it could also be modified by loading individual configuration profiles

Reply:
No, I was going to do a logging pass in a future PR 1) create a top-level logger module for entrypoints, 2) simplify api pkg/module logging, as really only the entrypoint needs the configured logger